pages/docs/metrics.mdx

```diff
@@ -125,7 +125,7 @@ To copy a metric's configuration into a metric tile, follow these steps:
 
 ## Metrics as code
```

Comment:
I feel like the `Metrics in dashboards` section above this should probably be updated as well to reference linked metric tiles.